proxies/docs/README.md

```diff
@@ -0,0 +1,349 @@
+
+<h1 id="devcycle-test-harness-proxy">DevCycle Test Harness - Proxy v1.0.0</h1>
```

Comment:
i was actually thinking this could be a top-level docs folder rather than being inside "proxies"